src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,51 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class FigureSupplier {
+    private final int figuresCount = 3;
```

Comment:
constants should be uppercased

Reply:
When I do FIGURES_COUNT I got mistakes with maven test

Reply:
![Uploading Screenshot 2023-10-10 at 21.05.36.png…]()

Reply:
<img width="1440" alt="Screenshot 2023-10-10 at 21 05 36" src="https://github.com/mate-academy/jv-oop-advanced/assets/60583206/819f9489-115e-4611-8d50-68eb7074a22f">